packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx

```diff
@@ -58,16 +74,15 @@ export class GuxAvatar {
       <div
         class={{
           'gux-avatar': true,
-          [`gux-${this.size}`]: true
+          [`gux-${this.size}`]: true,
+          [`gux-accent-${this.getAccent()}`]: true
         }}
       >
-        <div class="gux-content">
-          <slot name="image">
-            {!this.root.querySelector('[slot="image"]') && (
-              <abbr title={this.name}>{this.generateInitials()}</abbr>
-            )}
-          </slot>
-        </div>
+        <slot name="image">
+          {!this.root.querySelector('[slot="image"]') && (
+            <abbr title={this.name}>{this.generateInitials()}</abbr>
+          )}
+        </slot>
```

Comment:
I think this could just be ,

`       <slot name="image">
        <abbr title={this.name}>{this.generateInitials()}</abbr>
</slot>
          `
and if no content is provided to the image slot it would you the abbreviation as a fallback I think ?

Reply:
Yeah, I think that's better. I've updated it now.